libpldmresponder/platform_config.hpp

```diff
@@ -46,6 +46,17 @@ class Handler : public CmdHandler
     void registerSystemTypeCallback(SystemTypeCallback callback);
 
   private:
+    /** @brief Interface to get the first available directory
+     *         available from the received list
+     *
+     *  @param[in] dirNames - Getting system names from remote application
```

Comment:
Change the order of params to dirPath and dirNames

Reply:
corrected as suggestion.